app/views/articles/_full_comment_area.html.erb

```diff
@@ -2,8 +2,13 @@
   <div id="comments" data-updated-at="<%= Time.current %>">
     <% if @article.show_comments %>
       <div class="comments-container-container">
+        <%# TODO: Fix margins, vertical centering %>
+        <div class="flex justify-between ml-7 mr-8">
```

Comment:
```suggestion
        <div class="flex justify-between items-center mx-2 s:mx-7 mb-2">
```

- `items-center` to align two elements vertically,
- `mb-2` to add a little bit spacing below,
- `mx-2 s:mx-7`  instead of `ml-7 mr-8` so for mobile it will be `mx-2` and above mobile, it will be `mx-7`. `mx-*` is equivalent of `ml-* mr-*`.

That last change is to address some weird sizing we have in this section... More in another comment.

Reply:
I had used `items-center` but something with a higher specificity seemed to be messing with it. I wasn't aware of `mx-*` or `s:m*-*`. Thanks for pointing me to those.